src/mmw/js/src/draw/templates/aoiUpload.html

```diff
@@ -1 +1,22 @@
-<span id="dropbox">Drop a .shp file here.</span>
+<ul class="draw-pane-list">
+  <li class="draw-pane-list-item" role="presentation">
+    <div>
+        <span class="draw-button-title pull-left"
+              id="dropbox">
+            Drag and drop a file
+            <ul class="draw-pane-description-list">
+                <li>Must be a shapefile (.zip) or geojson</li>
+                <li>Less than 200 MB</li>
```

Comment:
I suggested this, but I think we should remove it until we actually implement a limit.